validators/roles.js

```diff
@@ -0,0 +1,15 @@
+const userRoles = require("../configuration/user-roles")
+
+function doesRoleExist(role) {
+    for (const key in userRoles) {
```

Comment:
You can use "some" function for this. And also you can merge password and roles validatior in one file. utils/validators.js